### ISSUE TITLE
Décrochage récurrent de nœud

### ISSUE THREAD

**Issue by LeoBoudet** [duniter/duniter#1210]
Je ne sais pas si cela est lié à la version 1.6.14 mais depuis que je l'ai installée, mon nœud décroche sans arrêt et ne parvient pas à raccrocher les wagons. Régulièrement il perd toutes ses connexions aux autres nœuds et ne les récupère pas de lui-même.
Séries de messages type :

**WS2P: Could not connect to peer D3krfq6J using `WS2P 77.152.31.154 20900: WS2P connection timeout`**

_Start/stop server_ n'améliorent pas le problème. Le dernier _reset data_ n'a rien donné non plus...

[duniter.log](https://github.com/duniter/duniter/files/1484498/duniter.log)

**Comment by c-geek** [comment]
C'est étrange, car j'ai eu une problématique similaire hier.

Ma solution : stopper Duniter pendant plus de 2 minutes (toutes les instances que tu peux avoir). Puis, le relancer.

**Comment by LeoBoudet** [comment]
Ça ne fonctionne pas dans mon cas.
Un autre _reset data_ n'a rien donné non plus.
Je vais tout réinstaller, on verra bien.

**Comment by LeoBoudet** [comment]
La réinstallation n'a rien donné. Je reste planté. Une connexion trop faiblarde ?

**Comment by M5oul** [comment]
I also have those logs of WS2P disconnections even on a stable and an unstable connections.

**Comment by LeoBoudet** [comment]
Oui, je les ai aussi lorsque mon nœud est connecté mais pas autant : là ils concernent tous les nœuds que j'essaie de connecter, sans exception.

**Comment by c-geek** [comment]
Leo : sinon, stoppe Duniter et supprime son home (~/.config/duniter) et relance-le. Il faudra tout resynchroniser et reparamétrer (notamment mettre ta clé de membre), mais ça pourrait jouer.

**Comment by LeoBoudet** [comment]
Déjà testé, et plusieurs fois, sur deux machines différentes, en clé membre et en miroir.
Toujours le même problème...

**Comment by c-geek** [comment]
Et avec une ancienne version ? (celle qui marchait) Ça permettrait d'incriminer une modification particulière.
